.azure/azure-pipeline.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     displayName: 'Install .NET Core SDK'
     inputs:
       packageType: 'sdk'
-      useGlobalJson: true
+      version: 3.1.300
```

Comment:
Have Azure Pipelines install the latest version of the SDK without reading it from global.json

Reply:
I'd rather the build use the version specified in global.json since that is what's used when developing locally.